src/components/Login/LoginComponent.js

```diff
@@ -21,9 +21,7 @@ class Login extends Component {
     handleLogin(event) {
         this.props.loginUser({ username: this.username.value, password: this.password.value });
         event.preventDefault();
-       console.log("-----------------");
-        console.log("role in handlelogin "+this.props.auth.userRole )
-       let uRole=localStorage.getItem('userRole');
```

Comment:
You can even dismiss this commit and use the localStorage  global variable as long as you handle parallelism correctly